### DISCUSSION TITLE
The model does not recognize the paper！

### DISCUSSION THREAD

**Discussion by EiserLiu** [Mayuresh1611/Paper-Piano#6] [Q&A]
Why can't I recognize the paper when I turn on the camera, I can't draw the piano, and it's no problem to recognize the hand
![a](https://github.com/Mayuresh1611/Paper-Piano/assets/140388451/064a2411-6932-4af7-b594-e8a08a3ef9c3)
![b](https://github.com/Mayuresh1611/Paper-Piano/assets/140388451/0f6bbc06-f732-4aa4-a56a-b3fc72ed4954)

**Comment by Mayuresh1611** [comment]
perhaps the algorithm is finding too much distractions with the edges of paper, also try to keep the drawn rectangles as flat as possible, then only the algo will draw the mesh.

**Reply by EiserLiu** [reply]
Okay, I've got time to print a perfect rectangle
